### PR TITLE
Add branch portafolio with README, html and css updated

### PR DESCRIPTION
**Pull request by DarioAlessioR**
Project portfolio branch updated with README, HTML and CSS files created locally with Visual Studio Code.

Linters were locally applied and passed. Errors were solved.

### CODE REVIEW

File: PortafolioMobile.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            <li><a href="#portafolio">Portafolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#email"><img class="email" src="Icon-Mail.png" alt="email"></a></li>
+          </ul>
+        </nav>
+
+      <div class="menu">
+        <img src="Union.png" alt="menu">
+      </div>
```
Comment (chrystalme) [inline PortafolioMobile.html]: Kindly remove design images as `` and replace them with background images.

File: PortafolioMobile.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    </header>
+    <section id="portafolio">
+      <img class="circle1" src="ball.png" alt="ball left">
+      <img class="circle2" src="ball.png" alt="ball right">
```
Comment (chrystalme) [inline PortafolioMobile.html]: Kindly remove design images as `` and replace them with background images.

File: PortafolioMobile.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      <h1 class="name">Dario Alessio</h1>
+      <h2 class="profession">Software Developer</h2>
+      
+      <h4 class="desc">I'm a software developer and can help
+         your software projects go ahead and finish on time.
+          Look through my portafolio and you'll see what I can do.
+           If you have any doubt or are interested in explore
+            options about how I can contribute to your projects,
+          please don't hesitate to contact me.</h4>
+
```
Comment (chrystalme) [inline PortafolioMobile.html]: Use `<p>` for sentences in your design for sematic purposes and better SEO optimisation in your web page.

File: PortafolioMobile.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      <img class="innerItems" src="VectorGit.png" alt="">
+      <img class="disableIN" src="DisabledIN.png" alt="">
+      <img class="innerItems" src="VectorHand.png" alt="">
+      <img class="innerItems" src="VectorTwitter.png" alt="">
+      <img class="innerItems" src="VectorM.png"  alt="">
+      <img class="Group55b" src="Group55.png" alt="">
+      <h6>Work</h6>
+    </div>
+    </section>
+  </body>
```
Comment (chrystalme) [inline PortafolioMobile.html]: Kindly use an unordered list - `<ul>` `<li>` for the social icons and make each icon a link `<a>`. Also, it is a good practise to use  fonts like [fontawesome](https://fontawesome.com/) which are lighter than images.

**Review by chrystalme — CHANGES_REQUESTED** [review]
# Hi @DarioAlessioR , 

You have done a good job so far! 👏🏽 👏🏽 
However, there are some issues that you still need to work on to prepare your project for further evaluation.

## Suggested changes ♻️ 

- [x]  You have not customised the Microverse Readme.md template. Kindly, customise it and include your social media handles and appropriate links
- [x] You are not using margins to create space around elements.

|           Your design                     |                   Figma Design          |
|---|---|
|Web capture_22-4-2022_95220_127 0 0 1|image

- [x] Kindly make sure you are using the exact color from the figma design.
- [ ] I have also included some inline comments fir you to take care of. Kindly attend to these comments. 



## Optional Changes

You can also consider:

- [x] Putting all your images in an images folder. This makes your work more organised and professional looking. 
- [x] Make frequent commits

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DarioAlessioR** [comment]
Changes asked by @chrystalme have been done. Thank you so much for your kind review.

File: README.md [hunk 5]
```diff
@@ -1,60 +1,36 @@
...
-## Built With
+This is the first part of the whole project and so it includes only the homepage for mobile devices.
 
-- Major languages
-- Frameworks
-- Technologies used
+Project will be updated in next weeks with more information about the completed projects that will form part of the portafolio.
 
-## Live Demo (if available)
+The project hs been done following template # 4 of Figma options provided by Microverse.
```
Comment (youmari) [inline README.md]: As the previous reviewer asked you, you should only customize it by adding your information so please try to leave all the important header from the[ template README](https://github.com/microverseinc/readme-template) and add you information for every important header

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @DarioAlessioR , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by DarioAlessioR** [comment]
Changes suggested by @youmari have been applied to the README file.

Thank you in advance for your kind review.

File: README.md [hunk 6]
```diff
@@ -1,73 +1,63 @@
...
+Project Name: Dario Alessio's Portfolio
 
-> Description the project.
+Description the project: 
+This is a page for mobile devices create to include professional information regarding technical and soft skills and abilities to inform potential employers.
 
+This is the first part of the whole project and so it includes only the homepage for mobile devices.
 
-## Built With
+Project will be updated in next weeks with more information about the completed projects that will form part of the portafolio.
```
Comment (Wes-Isaac) [inline README.md:10]: - [ ] [OPTIONAL] As you can see here the titles have a # before them, kindly add that so it will look more professional.

**Review by Wes-Isaac — APPROVED** [review]
Hi @DarioAlessioR, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DarioAlessioR** [comment]
Merge branch portfolio after receiving approval from third Microverse review by @Wes-Isaac.

Thank you for your kind review @Wes-Isaac @chrystalme @youmari

**Comment by Wes-Isaac** [comment]
you're welcome @DarioAlessioR.

**Pull request merged**